.github/workflows/lint-and-test.yml

```diff
@@ -0,0 +1,83 @@
+name: Lint and test
```

Comment:
Apparently this file uses pico match for filtering files.

Refs: https://github.com/dorny/paths-filter#notes

Reply:
Fixed.